packages/core/src/browser/test/jsdom-environment.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+const JestJSDOMEnvironment = require('jest-environment-jsdom');
```

Comment:
So `jest-environment-jsdom` would make so that we do not need the dependency to `jsdom` explicitly in the `package.json` right? Can it be removed?